.github/workflows/pytest.yml

```diff
@@ -44,4 +45,4 @@ jobs:
         python -m flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        python -m pytest
+        python -m pytest --doctest-modules
```

Comment:
Could you use a pytest.ini as in paderbox=

Reply:
Done